awswrangler/dynamodb/_utils.py

```diff
@@ -4,15 +4,18 @@
 from typing import Any, Dict, Iterator, List, Mapping, Optional, Union
 
 import boto3
+from boto3.dynamodb.types import TypeSerializer
 from botocore.exceptions import ClientError
 
 from awswrangler import _utils, exceptions
 from awswrangler._config import apply_configs
+from awswrangler.annotations import Deprecated
 
 _logger: logging.Logger = logging.getLogger(__name__)
 
 
 @apply_configs
+@Deprecated
 def get_table(
```

Comment:
What about the usages of `get_table`? `delete_items` seems to be using it.

Reply:
Agreed - there is a reason `Table` is so convenient and it feels like we are trying to recreate it from scratch. Deprecating the resource in the future wouldn't be a breaking change so it can wait